llm_on_ray/common/optimizer/group_optimizer.py

```diff
@@ -1,5 +1,5 @@
 import torch  # noqa: F401
-from .optimizer import Optimizer
+from llm_on_ray.common.optimizer.optimizer import Optimizer
```

Comment:
should expose Optimizer in __all__ of __init__.py so that the code can be written in below style:
```suggestion
from llm_on_ray.common.optimizer import Optimizer
```